### PR TITLE
docs: typo in  `spatial_shader.rst`

### PR DESCRIPTION
**Pull request by alitnk**
Not 100% sure if this is a typo, but looks like one. Thank you.

### CODE REVIEW

**Review by clayjohn — APPROVED** [review]

File: tutorials/shaders/shader_reference/spatial_shader.rst [hunk 1]
```diff
@@ -254,7 +254,7 @@ these properties, and if you don't write to them, Godot will optimize away the c
 | in vec4 **FRAGCOORD**                  | Coordinate of pixel center in screen space. ``xy`` specifies position in window, ``z``           |
 |                                        | specifies fragment depth if ``DEPTH`` is not used. Origin is lower-left.                         |
 +----------------------------------------+--------------------------------------------------------------------------------------------------+
-| in bool **FRONT_FACING**               | ``true`` if current face if front face.                                                          |
+| in bool **FRONT_FACING**               | ``true`` if current face is front face.                                                          |
```
Comment (AThousandShips) [inline tutorials/shaders/shader_reference/spatial_shader.rst:257]:
```suggestion
| in bool **FRONT_FACING**               | ``true`` if current face is front facing.                                                        |
```
Or
```suggestion
| in bool **FRONT_FACING**               | ``true`` if current face is a front face.                                                        |
```

Comment (alitnk) [reply tutorials/shaders/shader_reference/spatial_shader.rst]: Was thinking about making the same change, actually. Thank you!

**Review by skyace65 — APPROVED** [review]

**Comment by skyace65** [comment]
Thanks! And congrats on your first merged PR!

**Comment by mhilbrunner** [comment]
Cherry-picked to 4.2 in https://github.com/godotengine/godot-docs/pull/9647.

**Pull request merged**